dev/AppLifecycle/EncodedLaunchExecuteCommand.cpp

```diff
@@ -2,13 +2,14 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 #include "pch.h"
+#include "ExtensionContract.h"
 #include "EncodedLaunchExecuteCommand.h"
 #include "ActivationRegistrationManager.h"
 
 using namespace winrt;
 using namespace winrt::Windows::Foundation;
 
-namespace winrt::Microsoft::ApplicationModel::Activation::implementation
+namespace winrt::Microsoft::Windows::AppLifecycle::implementation
```

Comment:
>Microsoft::Windows::AppLifecycle [](http://example.com/codeflow?start=17&length=32)

You'll also need to update your namespace at...

build\AzurePipelinesTemplates\ProjectReunion-CreateNugetPackage-Job.yml
build\CopyFilesToStagingDir.ps1
build\NuSpecs\AppxManifest.xml
build\NuSpecs\Intellisense\Microsoft.ApplicationModel.Activation.xml
build\NuSpecs\ProjectReunion-Nuget-Native.WinRt.props
test\DynamicDependency\data\Microsoft.ProjectReunion.Framework\appxmanifest.xml

Until those are corrected some of those will fail the pipeline fatally, and some will fail the pipeline nonfatally - as in, an error in a powershell script will be ignored and PS moves on, resulting in the pipeline end status says OK/green, but it's actually incomplete and anyone using Reunion NuGet and MSIX will get build + runtime errors (yeah, known issue. Being worked on)

FYI regex I used to find all references to the namespace

Microsoft..?ApplicationModel..?Activation

as a case-insenstive regex across all files in ProjectReunion

(and the internal build pipeline, which found no hits. As it shouldn't; that's all data driven by what comes out of the ProjectReunion repo and build pipeline)

Reply:
BTW
```
DIR *activation* /s/b
```
only turns up 1 hit build\NuSpecs\Intellisense\Microsoft.ApplicationModel.Activation.xml (mentioned above). You'll need to rename that file when you rename the content of the rest to point to the right new filename